### PR TITLE
fix: Modify files in the release archive

### PR DESCRIPTION
**Pull request by mikolajb**
**What problem does this PR solve?**:
It looks we have to modify a file while creating an archive.

**Which issue(s) does this PR fix?**:
https://jira.nutanix.com/browse/D2IQ-100557

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Checklist**

- [ ] If the PR adds a version bump, ensure there is no breaking change in Licensing model (or NA).
- [ ] If a chart is changed or app configuration is significantly changed, the chart version is correctly incremented (so that apps are not automatically upgraded from a previous version of DKP).

### CODE REVIEW

File: make/release.mk [hunk 1]
```diff
@@ -15,6 +15,7 @@ release:
 								  ":(exclude)common/helm-repositories/ai-navigator-repos.yaml" \
 								  ":(exclude)services/ai-navigator-app" \
 								  ":(exclude)services/ai-navigator-cluster-info-agent"
+	./hack/hack/build-archive.sh $(ARCHIVE_NAME)
```
Comment (mhrabovcin) [inline make/release.mk:18]:
```suggestion
	./hack/build-archive.sh $(ARCHIVE_NAME)
```

Is there a duplicate `hack` in the path?

**Comment by mhrabovcin** [comment]
Would it be possible to add a patch to `./common/airgapped` to not include these repositories instead of rebuilding tar archive. We could target a dedicated label on `HelmRelease` resource instead of hardcoding the AI navigator specific charts.

Example: https://github.com/kubernetes-sigs/kustomize/issues/1593#issuecomment-1545727117

**Comment by mikolajb** [comment]
> Would it be possible to add a patch to `./common/airgapped` to not include these repositories instead of rebuilding tar archive. We could target a dedicated label on `HelmRelease` resource instead of hardcoding the AI navigator specific charts.
> 
> Example: [kubernetes-sigs/kustomize#1593 (comment)](https://github.com/kubernetes-sigs/kustomize/issues/1593#issuecomment-1545727117)

I think the idea is to not to have any trace of `ai` in our airgapped bundles. In other words - we have to remove it in build-time not run-time. Let's also hear from @gracedo as she authored the [previous changes](https://github.com/mesosphere/kommander-applications/pull/2135).

**Review by mhrabovcin — APPROVED** [review]

**Review by msdolbey — APPROVED** [review]
👍 
do bundle up

**Pull request merged**